proposals/infra/INF-0002-root-signature-in-clang.md

```diff
@@ -466,6 +456,24 @@ elimation has completed.
 Most values like ShaderVisibility/ParameterType are covered by syntactical checks in Sema.
 Only list special rule here.
 
+- DESCRIPTOR_RANGE_FLAGS for Sampler
```

Comment:
I think that what you're trying to say is during validation, descriptor ranges flags will be checked, and here is the list of valid flags for Sampler descriptor ranges, and the valid flags for UAV, CBV and SRC descriptor ranges.

These specs / proposals need to be written so that they can be read by someone who hasn't had any of the conversations and design meetings and reviews of existing code that you've had, so it is not enough to write them in note form.

Reply:
Updated.
Hope it make things clearer.